.github/workflows/regression.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Regression Testing
+
+on:
+  deployment_status:
+
+defaults:
+  run:
```

Comment:
In general, do we need this file if only kicking off locally right now?

Reply:
Its going to be coming into play in the next sprint so...maybe?

Reply:
Removed here: 517ffaf9fcf5e76929e9409a8e8dca027af75ac5